tests/testthat/test-bind-cache.R

```diff
@@ -1209,7 +1209,11 @@ test_that("Custom render functions that call exprToFunction", {
 
 test_that("Some render functions can't be cached", {
   m <- cachem::cache_mem()
-  expect_error(renderDataTable({ cars }) %>% bindCache(1, cache = m))
+  expect_error(
+    lifecycle::expect_deprecated(
+      renderDataTable({ cars }) %>% bindCache(1, cache = m)
+    )
+  )
```

Comment:
I am worried the checks are inverted. Ex: If the `lifecycle` expectation throws, an error will be caught, and that is a success.

Is it possible to swap the two expectations? (`lifecycle` on the outside and `expect_error` on the inside)

Reply:
Yup, thanks for pointing this out!